include/Util/Image.hpp

```diff
@@ -29,8 +28,10 @@ class Image : public Core::Drawable {
      * @brief Constructor that takes a file path to the image.
      *
      * @param filepath The file path to the image.
+     * @param useAA Flag indicating whether anti-aliasing should be enabled
+     * (default is true).
      */
-    explicit Image(const std::string &filepath);
+    Image(const std::string &filepath, bool useAA = true);
```

Comment:
why do you remove the `explicit`

Reply:
because there are 2 parameters and C++ can't implicit convert in this case